TundraTD/Assets/Scripts/SceneManagement/LevelsScene.cs

```diff
@@ -0,0 +1,52 @@
+﻿using UnityEngine;
+using UnityEngine.SceneManagement;
+using UnityEngine.UI;
+
+namespace SceneManagement
+{
+    public class LevelsScene : MonoBehaviour
+    {
+        [SerializeField] private RectTransform buttonsHolder;
+        [SerializeField] private Button levelButtonPrefab;
+        private int _levelsAmount;
+        private int _currentLevelBindingIndex = 1;
+        private const int LevelsBindingStartIndex = 2;
```

Comment:
Nit: I'd suggest to move constants in top of the class definition. It would make constants management easier